ts/IterableInboxMessageDisplay.tsx

```diff
@@ -46,13 +50,59 @@ const IterableInboxMessageDisplay = ({
    returnButtonContainer = {...returnButtonContainer, marginTop: Platform.OS === 'android' ? 0 : 40}
    returnButtonContainer = (!isPortrait) ? {...returnButtonContainer, marginTop: 10} : returnButtonContainer
 
+   let JS = `
+      const links = document.querySelectorAll('a')
+      links.forEach(link => {
+         if(link.href === "iterable://dismiss") {
+            link.class = "dismiss"   
+         }
+
+         if(link.href === "iterable://delete") {
+            link.class = "delete"
+         }
+
+         link.addEventListener("click", () => {
+            if(link.class === "dismiss") {
```

Comment:
I couldn't figure out why was it necessary to have link.class = "dismiss" or "delete" when we could just check it in if condition here? 
Is link.class being used else where?

Reply:
I added the class label since I have to change the href to deactivate the HTML link. Without the deactivation, the HTML link tries to open the iterable action. By changing the href to "javascript: void(0)", the link will redirect to stay on the same page. On a future PR, I will be working on the more generalized case by incorporating internal links into the logic.